src/ex1/style_ex1.css

```diff
@@ -0,0 +1,173 @@
+body {
+  font-family: "Dosis", sans-serif;
+  background-color: rgb(153, 203, 205);
+}
+
+.white-box {
+  position: relative;
+  margin: 0 auto;
+  width: 400px;
+  height: 600px;
+  background-color: white;
+  border-radius: 1%;
+}
+
+.main-header {
+  margin-left: 25px;
+  padding-top: 15px;
+}
+
+.task-input-wrapper {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 15px;
+}
+
+.task-input {
+  margin-left: 25px;
+  padding: 10px;
+  width: 60%;
+  border: 1px solid rgb(197, 196, 196);
+  border-radius: 3px;
+  outline: none;
+}
+
+.task-button {
+  margin-right: 25px;
+  font-size: medium;
+  padding: 9px 13px;
+  width: 20%;
+  background-color: rgb(153, 90, 212);
+  color: white;
+  border: none;
+  border-radius: 3px;
+}
+
+.task-button:hover {
+  cursor: pointer;
+  background-color: cadetblue;
+}
+
+.task-list {
+  display: flex;
+  flex-direction: column;
```

Comment:
Visual bug: There is no scroll, so if I add too many tasks it overflows

![image](https://user-images.githubusercontent.com/5465198/170648698-ca9ebb03-2800-4a57-8875-fe96a6cf6aa6.png)